docs/source/routing/observability/telemetry/instrumentation/events.mdx

```diff
@@ -202,9 +202,25 @@ telemetry:
               response_header: "x-my-header"
 ```
 
-## Event configuration example
+## Event configuration reference
+
```

Comment:
Maybe it's already obvious, but could it be helpful to add one intro sentence to explain how to use this table? (I know this table was just moved, not created, in this PR, but couldn't help but wonder.) :)